src/common/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if(OQS_USE_AES_OPENSSL)
     set(AES_IMPL aes/aes_ossl.c)
 else()
    set(AES_IMPL aes/aes.c aes/aes_c.c)
-    if (OQS_USE_AES_INSTRUCTIONS)
+   if (OQS_PORTABLE_X86_64_BUILD OR OQS_USE_AES_INSTRUCTIONS)
```

Comment:
Wouldn't it be sufficient (even more correct) to add `ARCH_X86_64 AND  OQS_USE_AES_INSTRUCTIONS`? Presence/absence of this code should be independent of OQS_PORTABLE_BUILD.